### PR TITLE
catch occasional exception

### PR DESCRIPTION
**Pull request by passchaos**
Somtimes, alert created from app may trigger exception, this patch catch it.

### CODE REVIEW

File: Classes/UIAutomationHelper.m [hunk 1]
```diff
@@ -136,7 +136,16 @@ - (BOOL)_alertIsValidAndVisible:(UIAAlert *)alert;
     // Translate the __NSCFBoolean into a vanilla BOOL.
     // See https://www.bignerdranch.com/blog/bools-sharp-corners/ for more details.
     
-    BOOL visible = [[alert valueForKeyPath:@"isVisible"] boolValue];
+    BOOL visible = NO;
+    
+    @try {
+        visible = [[alert valueForKeyPath:@"isVisible"] boolValue];
+    } @catch (NSException *exception) {
+        visible = NO;
```
Comment (phatmann) [inline Classes/UIAutomationHelper.m]: Since visible is already NO, we don't need this line.

File: Classes/UIAutomationHelper.m [hunk 2]
```diff
@@ -136,7 +136,16 @@ - (BOOL)_alertIsValidAndVisible:(UIAAlert *)alert;
...
+    } @finally {
```
Comment (phatmann) [inline Classes/UIAutomationHelper.m]: Let's remove the empty `@finally` clause.

**Comment by phatmann** [comment]
Thanks for the PR. I requested a few minor changes before I merge it.

**Comment by passchaos** [comment]
Sorry, I just want to conform with "linkAutomationFramework", update yet!

**Comment by phatmann** [comment]
Thanks, all merged.

**Pull request merged**